docs-conceptual/azps-6.4.0/what-is-azure-powershell.md

```diff
@@ -0,0 +1,55 @@
+---
+title: What is Azure PowerShell
+description: This article is an introduction to Azure PowerShell and its features.
+ms.devlang: powershell
+ms.topic: conceptual
+ms.date: 09/13/2021
+ms.custom: devx-track-azurepowershell
+ms.service: azure-powershell
+---
+
+# What is Azure PowerShell?
+
+Azure PowerShell is a set of cmdlets for managing Azure resources directly from PowerShell. Azure
+PowerShell is designed to make it easy to learn and get started with, but provides powerful features
+for automation.
+
+## The Az PowerShell module
+
+> [!IMPORTANT]
+> The Az PowerShell module is the recommended PowerShell module for managing Azure resources on all
+> platforms.
+
+Introduced in 2018, the Az PowerShell module is based on the .NET Standard, and works with
```

Comment:
I'm concerned that the introduction date may be misinterpreted. Since then we have changed some fundamental components like ADAL -> MSAL.

Maybe we should have a section about the fact this is a wrapper module.